### PR TITLE
OMIM ingest

### LINKED ISSUE #3009: OMIM: some missing terms that were erroneously filtered
Sorry these may take a while, but I am confident these are all. Do you use your text editor to add them all? How long will it take you do these? Estimate.

```
[Term]
id: MONDO:0021421
name: carcinoid tumors, intestinal
synonym: "CARCINOID TUMORS, INTESTINAL" RELATED  [OMIM:114900]
xref: UMLS:C0349535 {source="OMIM:114900"}
is_a: MONDO:0003847
xref: OMIM:114900 {source="MONDO:equivalentTo"} ! carcinoid tumors, intestinal

[Term]
id: MONDO:0025351
name: multiple congenital anomalies-neurodevelopmental syndrome, x-linked
synonym: "Linked Syndrome" RELATED  [OMIM:301056]
synonym: "MULTIPLE CONGENITAL ANOMALIES-NEURODEVELOPMENTAL SYNDROME, X-LINKED; MCAND" RELATED  [OMIM:301056]
is_a: MONDO:0003847
xref: OMIM:301056 {source="MONDO:equivalentTo"} ! multiple congenital anomalies-neurodevelopmental syndrome, x-linked

[Term]
id: MONDO:0025356
name: azoospermia, obstructive, with nephrolithiasis
synonym: "AZOOSPERMIA, OBSTRUCTIVE, WITH NEPHROLITHIASIS; OAZON" RELATED  [OMIM:301060]
is_a: MONDO:0003847
xref: OMIM:301060 {source="MONDO:equivalentTo"} ! azoospermia, obstructive, with nephrolithiasis

[Term]
id: MONDO:0025622
name: charcot-marie-tooth disease, axonal, mitochondrial form, 1
synonym: "CHARCOT-MARIE-TOOTH DISEASE, AXONAL, MITOCHONDRIAL FORM, 1; CMTMA1" RELATED  [OMIM:500013]
is_a: MONDO:0003847
xref: OMIM:500013 {source="MONDO:equivalentTo"} ! charcot-marie-tooth disease, axonal, mitochondrial form, 1

[Term]
id: MONDO:0030835
name: developmental delay, impaired growth, dysmorphic facies, and axonal neuropathy
synonym: "DEVELOPMENTAL DELAY, IMPAIRED GROWTH, DYSMORPHIC FACIES, AND AXONAL NEUROPATHY; DIGFAN" RELATED  [OMIM:619090]
is_a: MONDO:0003847
xref: OMIM:619090 {source="MONDO:equivalentTo"} ! developmental delay, impaired growth, dysmorphic facies, and axonal neuropathy

[Term]
id: MONDO:0030836
name: neurodevelopmental disorder with microcephaly, impaired language, and gait abnormalities
synonym: "NEURODEVELOPMENTAL DISORDER WITH MICROCEPHALY, IMPAIRED LANGUAGE, AND GAIT ABNORMALITIES; NEDMILG" RELATED  [OMIM:619091]
is_a: MONDO:0003847
xref: OMIM:619091 {source="MONDO:equivalentTo"} ! neurodevelopmental disorder with microcephaly, impaired language, and gait abnormalities

[Term]
id: MONDO:0030837
name: neurodevelopmental disorder with microcephaly, impaired language, epilepsy, and gait abnormalities
synonym: "NEURODEVELOPMENTAL DISORDER WITH MICROCEPHALY, IMPAIRED LANGUAGE, EPILEPSY, AND GAIT ABNORMALITIES; NEDMILEG" RELATED  [OMIM:619092]
is_a: MONDO:0003847
xref: OMIM:619092 {source="MONDO:equivalentTo"} ! neurodevelopmental disorder with microcephaly, impaired language, epilepsy, and gait abnormalities

[Term]
id: MONDO:0030849
name: intellectual developmental disorder with speech delay and axonal peripheral neuropathy
synonym: "INTELLECTUAL DEVELOPMENTAL DISORDER WITH SPEECH DELAY AND AXONAL PERIPHERAL NEUROPATHY; IDDSAPN" RELATED  [OMIM:619099]
is_a: MONDO:0003847
xref: OMIM:619099 {source="MONDO:equivalentTo"} ! intellectual developmental disorder with speech delay and axonal peripheral neuropathy

[Term]
id: MONDO:0030852
name: neurodevelopmental disorder with dysmorphic facies, sleep disturbance, and brain abnormalities
synonym: "NEURODEVELOPMENTAL DISORDER WITH DYSMORPHIC FACIES, SLEEP DISTURBANCE, AND BRAIN ABNORMALITIES; NEDFASB" RELATED  [OMIM:619103]
is_a: MONDO:0003847
xref: OMIM:619103 {source="MONDO:equivalentTo"} ! neurodevelopmental disorder with dysmorphic facies, sleep disturbance, and brain abnormalities

[Term]
id: MONDO:0030859
name: coach syndrome 2
synonym: "COACH SYNDROME 2; COACH2" RELATED  [OMIM:619111]
is_a: MONDO:0003847
xref: OMIM:619111 {source="MONDO:equivalentTo"} ! coach syndrome 2

[Term]
id: MONDO:0030860
name: neuronopathy, distal hereditary motor, type 5c
synonym: "Dhmn5C" RELATED  [OMIM:619112]
synonym: "NEURONOPATHY, DISTAL HEREDITARY MOTOR, TYPE VC; HMN5C" RELATED  [OMIM:619112]
synonym: "Spinal Muscular Atrophy, Distal, Type 5C" RELATED  [OMIM:619112]
is_a: MONDO:0003847
xref: OMIM:619112 {source="MONDO:equivalentTo"} ! neuronopathy, distal hereditary motor, type 5c

[Term]
id: MONDO:0030862
name: coach syndrome 3
synonym: "COACH SYNDROME 3; COACH3" RELATED  [OMIM:619113]
is_a: MONDO:0003847
xref: OMIM:619113 {source="MONDO:equivalentTo"} ! coach syndrome 3

[Term]
id: MONDO:0030866
name: neurodevelopmental disorder with cardiomyopathy, spasticity, and brain abnormalities
synonym: "NEURODEVELOPMENTAL DISORDER WITH CARDIOMYOPATHY, SPASTICITY, AND BRAIN ABNORMALITIES; NEDCASB" RELATED  [OMIM:619121]
is_a: MONDO:0003847
xref: OMIM:619121 {source="MONDO:equivalentTo"} ! neurodevelopmental disorder with cardiomyopathy, spasticity, and brain abnormalities

[Term]
id: MONDO:0030871
name: vertebral hypersegmentation and orofacial anomalies
synonym: "VERTEBRAL HYPERSEGMENTATION AND OROFACIAL ANOMALIES; VHO" RELATED  [OMIM:619122]
is_a: MONDO:0003847
xref: OMIM:619122 {source="MONDO:equivalentTo"} ! vertebral hypersegmentation and orofacial anomalies

[Term]
id: MONDO:0030873
name: cardiofacioneurodevelopmental syndrome
synonym: "CARDIOFACIONEURODEVELOPMENTAL SYNDROME; CFNDS" RELATED  [OMIM:619123]
is_a: MONDO:0003847
xref: OMIM:619123 {source="MONDO:equivalentTo"} ! cardiofacioneurodevelopmental syndrome

[Term]
id: MONDO:0030878
name: kaya-barakat-masson syndrome
synonym: "KAYA-BARAKAT-MASSON SYNDROME; KABAMAS" RELATED  [OMIM:619125]
is_a: MONDO:0003847
xref: OMIM:619125 {source="MONDO:equivalentTo"} ! kaya-barakat-masson syndrome

[Term]
id: MONDO:0030880
name: mandibuloacral dysplasia progeroid syndrome
synonym: "MANDIBULOACRAL DYSPLASIA PROGEROID SYNDROME; MDPS" RELATED  [OMIM:619127]
is_a: MONDO:0003847
xref: OMIM:619127 {source="MONDO:equivalentTo"} ! mandibuloacral dysplasia progeroid syndrome

[Term]
id: MONDO:0030885
name: amyotrophic lateral sclerosis 26 with or without frontotemporal dementia
synonym: "AMYOTROPHIC LATERAL SCLEROSIS 26 WITH OR WITHOUT FRONTOTEMPORAL DEMENTIA; ALS26" RELATED  [OMIM:619133]
is_a: MONDO:0003847
xref: OMIM:619133 {source="MONDO:equivalentTo"} ! amyotrophic lateral sclerosis 26 with or without frontotemporal dementia

[Term]
id: MONDO:0030893
name: leukoencephalopathy, progressive, infantile-onset, with or without deafness
synonym: "LEUKOENCEPHALOPATHY, PROGRESSIVE, INFANTILE-ONSET, WITH OR WITHOUT DEAFNESS; LEPID" RELATED  [OMIM:619147]
is_a: MONDO:0003847
xref: OMIM:619147 {source="MONDO:equivalentTo"} ! leukoencephalopathy, progressive, infantile-onset, with or without deafness

[Term]
id: MONDO:0030896
name: chromosome 13q33-q34 deletion syndrome
synonym: "CHROMOSOME 13q33-q34 DELETION SYNDROME" RELATED  [OMIM:619148]
is_a: MONDO:0003847
xref: OMIM:619148 {source="MONDO:equivalentTo"} ! chromosome 13q33-q34 deletion syndrome

[Term]
id: MONDO:0030897
name: lessel-kreienkamp syndrome
synonym: "LESSEL-KREIENKAMP SYNDROME; LESKRES" RELATED  [OMIM:619149]
is_a: MONDO:0003847
xref: OMIM:619149 {source="MONDO:equivalentTo"} ! lessel-kreienkamp syndrome

[Term]
id: MONDO:0030900
name: intellectual developmental disorder with paroxysmal dyskinesia or seizures
synonym: "INTELLECTUAL DEVELOPMENTAL DISORDER WITH PAROXYSMAL DYSKINESIA OR SEIZURES; IDDPADS" RELATED  [OMIM:619150]
is_a: MONDO:0003847
xref: OMIM:619150 {source="MONDO:equivalentTo"} ! intellectual developmental disorder with paroxysmal dyskinesia or seizures

[Term]
id: MONDO:0030930
name: neurodevelopmental disorder with or without early-onset generalized epilepsy
synonym: "NEURODEVELOPMENTAL DISORDER WITH OR WITHOUT EARLY-ONSET GENERALIZED EPILEPSY; NEDEGE" RELATED  [OMIM:619157]
is_a: MONDO:0003847
xref: OMIM:619157 {source="MONDO:equivalentTo"} ! neurodevelopmental disorder with or without early-onset generalized epilepsy

[Term]
id: MONDO:0030947
name: neurodegeneration, childhood-onset, with hypotonia, respiratory insufficiency, and brain imaging abnormalities
synonym: "NEURODEGENERATION, CHILDHOOD-ONSET, WITH HYPOTONIA, RESPIRATORY INSUFFICIENCY, AND BRAIN IMAGING ABNORMALITIES; CONRIBA" RELATED  [OMIM:619173]
is_a: MONDO:0003847
xref: OMIM:619173 {source="MONDO:equivalentTo"} ! neurodegeneration, childhood-onset, with hypotonia, respiratory insufficiency, and brain imaging abnormalities

[Term]
id: MONDO:0030956
name: gastric adenocarcinoma and proximal polyposis of the stomach
synonym: "Fundic Gland Polyposis" RELATED  [OMIM:619182]
synonym: "GASTRIC ADENOCARCINOMA AND PROXIMAL POLYPOSIS OF THE STOMACH; GAPPS" RELATED  [OMIM:619182]
synonym: "Polyposis, Gastric" RELATED  [OMIM:619182]
is_a: MONDO:0003847
xref: OMIM:619182 {source="MONDO:equivalentTo"} ! gastric adenocarcinoma and proximal polyposis of the stomach

[Term]
id: MONDO:0030963
name: li-campeau syndrome
synonym: "LI-CAMPEAU SYNDROME; LICAS" RELATED  [OMIM:619189]
is_a: MONDO:0003847
xref: OMIM:619189 {source="MONDO:equivalentTo"} ! li-campeau syndrome

[Term]
id: MONDO:0030966
name: neurofacioskeletal syndrome with or without renal agenesis
synonym: "Neurodevelopmental Disorder With Corpus Callosum Agenesis, Craniofacial Dysmorphism, and Skeletal Anomalies, With or Without Renal Agenesis" RELATED  [OMIM:619194]
synonym: "NEUROFACIOSKELETAL SYNDROME WITH OR WITHOUT RENAL AGENESIS; NFSRA" RELATED  [OMIM:619194]
is_a: MONDO:0003847
xref: OMIM:619194 {source="MONDO:equivalentTo"} ! neurofacioskeletal syndrome with or without renal agenesis

[Term]
id: MONDO:0030967
name: deafness, congenital, and adult-onset progressive leukoencephalopathy
synonym: "DEAFNESS, CONGENITAL, AND ADULT-ONSET PROGRESSIVE LEUKOENCEPHALOPATHY; DEAPLE" RELATED  [OMIM:619196]
is_a: MONDO:0003847
xref: OMIM:619196 {source="MONDO:equivalentTo"} ! deafness, congenital, and adult-onset progressive leukoencephalopathy

[Term]
id: MONDO:0030976
name: oculomotor-abducens synkinesis
synonym: "OCULOMOTOR-ABDUCENS SYNKINESIS; OCABSN" RELATED  [OMIM:619215]
is_a: MONDO:0003847
xref: OMIM:619215 {source="MONDO:equivalentTo"} ! oculomotor-abducens synkinesis

[Term]
id: MONDO:0030977
name: neuropathy, hereditary motor, with myopathic features
synonym: "NEUROPATHY, HEREDITARY MOTOR, WITH MYOPATHIC FEATURES; HMNMYO" RELATED  [OMIM:619216]
is_a: MONDO:0003847
xref: OMIM:619216 {source="MONDO:equivalentTo"} ! neuropathy, hereditary motor, with myopathic features

[Term]
id: MONDO:0030978
name: endove syndrome, limb-only type
synonym: "ENDOVE SYNDROME, LIMB-ONLY TYPE; ENDOVESL" RELATED  [OMIM:619217]
synonym: "Mesomelia of Lower Extremities With Hand and Foot Anomalies" RELATED  [OMIM:619217]
is_a: MONDO:0003847
xref: OMIM:619217 {source="MONDO:equivalentTo"} ! endove syndrome, limb-only type

[Term]
id: MONDO:0030979
name: endove syndrome, limb-brain type
synonym: "ENDOVE SYNDROME, LIMB-BRAIN TYPE; ENDOVESLB" RELATED  [OMIM:619218]
synonym: "Mesomelia of Lower Extremities With Hand, Foot, and Brain Anomalies" RELATED  [OMIM:619218]
is_a: MONDO:0003847
xref: OMIM:619218 {source="MONDO:equivalentTo"} ! endove syndrome, limb-brain type

[Term]
id: MONDO:0030982
name: sulfide:quinone oxidoreductase deficiency
synonym: "SULFIDE:QUINONE OXIDOREDUCTASE DEFICIENCY; SQORD" RELATED  [OMIM:619221]
is_a: MONDO:0003847
xref: OMIM:619221 {source="MONDO:equivalentTo"} ! sulfide:quinone oxidoreductase deficiency

[Term]
id: MONDO:0030986
name: blistering, acantholytic, of oral and laryngeal mucosa
synonym: "BLISTERING, ACANTHOLYTIC, OF ORAL AND LARYNGEAL MUCOSA; ABOLM" RELATED  [OMIM:619226]
is_a: MONDO:0003847
xref: OMIM:619226 {source="MONDO:equivalentTo"} ! blistering, acantholytic, of oral and laryngeal mucosa

[Term]
id: MONDO:0030987
name: vertebral, cardiac, tracheoesophageal, renal, and limb defects
synonym: "Vcterl Syndrome" RELATED  [OMIM:619227]
synonym: "VERTEBRAL, CARDIAC, TRACHEOESOPHAGEAL, RENAL, AND LIMB DEFECTS; VCTERL" RELATED  [OMIM:619227]
is_a: MONDO:0003847
xref: OMIM:619227 {source="MONDO:equivalentTo"} ! vertebral, cardiac, tracheoesophageal, renal, and limb defects

[Term]
id: MONDO:0030988
name: developmental delay with dysmorphic facies and dental anomalies
synonym: "DEVELOPMENTAL DELAY WITH DYSMORPHIC FACIES AND DENTAL ANOMALIES; DEFDA" RELATED  [OMIM:619228]
is_a: MONDO:0003847
xref: OMIM:619228 {source="MONDO:equivalentTo"} ! developmental delay with dysmorphic facies and dental anomalies

[Term]
id: MONDO:0030990
name: kohlschutter-tonz syndrome-like
synonym: "KOHLSCHUTTER-TONZ SYNDROME-LIKE; KTZSL" RELATED  [OMIM:619229]
is_a: MONDO:0003847
xref: OMIM:619229 {source="MONDO:equivalentTo"} ! kohlschutter-tonz syndrome-like

[Term]
id: MONDO:0030991
name: bile acid conjugation defect 1
synonym: "BILE ACID CONJUGATION DEFECT 1; BACD1" RELATED  [OMIM:619232]
is_a: MONDO:0003847
xref: OMIM:619232 {source="MONDO:equivalentTo"} ! bile acid conjugation defect 1

[Term]
id: MONDO:0030992
name: short stature, oligodontia, dysmorphic facies, and motor delay
synonym: "SHORT STATURE, OLIGODONTIA, DYSMORPHIC FACIES, AND MOTOR DELAY; SOFM" RELATED  [OMIM:619234]
is_a: MONDO:0003847
xref: OMIM:619234 {source="MONDO:equivalentTo"} ! short stature, oligodontia, dysmorphic facies, and motor delay

[Term]
id: MONDO:0030994
name: neurodevelopmental disorder with or without autism or seizures
synonym: "NEURODEVELOPMENTAL DISORDER WITH OR WITHOUT AUTISM OR SEIZURES; NEDAUS" RELATED  [OMIM:619239]
is_a: MONDO:0003847
xref: OMIM:619239 {source="MONDO:equivalentTo"} ! neurodevelopmental disorder with or without autism or seizures

[Term]
id: MONDO:0030995
name: global developmental delay with speech and behavioral abnormalities
synonym: "GLOBAL DEVELOPMENTAL DELAY WITH SPEECH AND BEHAVIORAL ABNORMALITIES; GDSBA" RELATED  [OMIM:619243]
is_a: MONDO:0003847
xref: OMIM:619243 {source="MONDO:equivalentTo"} ! global developmental delay with speech and behavioral abnormalities

[Term]
id: MONDO:0030999
name: neurodevelopmental disorder with cerebral atrophy and variable facial dysmorphism
synonym: "NEURODEVELOPMENTAL DISORDER WITH CEREBRAL ATROPHY AND VARIABLE FACIAL DYSMORPHISM; NEDCAFD" RELATED  [OMIM:619244]
is_a: MONDO:0003847
xref: OMIM:619244 {source="MONDO:equivalentTo"} ! neurodevelopmental disorder with cerebral atrophy and variable facial dysmorphism

[Term]
id: MONDO:0031001
name: vitreoretinopathy with phalangeal epiphyseal dysplasia
synonym: "VITREORETINOPATHY WITH PHALANGEAL EPIPHYSEAL DYSPLASIA; VPED" RELATED  [OMIM:619248]
is_a: MONDO:0003847
xref: OMIM:619248 {source="MONDO:equivalentTo"} ! vitreoretinopathy with phalangeal epiphyseal dysplasia

[Term]
id: MONDO:0031002
name: baralle-macken syndrome
synonym: "BARALLE-MACKEN SYNDROME; BARMACS" RELATED  [OMIM:619255]
synonym: "Neurodevelopmental Disorder With Cataracts and Variable Microcephaly" RELATED  [OMIM:619255]
is_a: MONDO:0003847
xref: OMIM:619255 {source="MONDO:equivalentTo"} ! baralle-macken syndrome

[Term]
id: MONDO:0031006
name: neurodegeneration with ataxia and late-onset optic atrophy
synonym: "NEURODEGENERATION WITH ATAXIA AND LATE-ONSET OPTIC ATROPHY; NDAXOA" RELATED  [OMIM:619259]
is_a: MONDO:0003847
xref: OMIM:619259 {source="MONDO:equivalentTo"} ! neurodegeneration with ataxia and late-onset optic atrophy

[Term]
id: MONDO:0031007
name: spondyloepiphyseal dysplasia, sensorineural hearing loss, impaired intellectual development, and leber congenital amaurosis
synonym: "Shilca Syndrome" RELATED  [OMIM:619260]
synonym: "SPONDYLOEPIPHYSEAL DYSPLASIA, SENSORINEURAL HEARING LOSS, IMPAIRED INTELLECTUAL DEVELOPMENT, AND LEBER CONGENITAL AMAUROSIS; SHILCA" RELATED  [OMIM:619260]
is_a: MONDO:0003847
xref: OMIM:619260 {source="MONDO:equivalentTo"} ! spondyloepiphyseal dysplasia, sensorineural hearing loss, impaired intellectual development, and leber congenital amaurosis

[Term]
id: MONDO:0031008
name: nephrotic syndrome, type 24
synonym: "NEPHROTIC SYNDROME, TYPE 24; NPHS24" RELATED  [OMIM:619263]
is_a: MONDO:0003847
xref: OMIM:619263 {source="MONDO:equivalentTo"} ! nephrotic syndrome, type 24

[Term]
id: MONDO:0031011
name: neurodevelopmental disorder with dysmorphic facies and variable seizures
synonym: "NEURODEVELOPMENTAL DISORDER WITH DYSMORPHIC FACIES AND VARIABLE SEIZURES; NEDDFAS" RELATED  [OMIM:619264]
is_a: MONDO:0003847
xref: OMIM:619264 {source="MONDO:equivalentTo"} ! neurodevelopmental disorder with dysmorphic facies and variable seizures
```

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #3009

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -410401,6 +410426,13 @@ intersection_of: MONDO:0005249 ! pneumonia
 intersection_of: MONDO:0005550 ! infectious disease
 intersection_of: disease_has_infectious_agent NCBITaxon:810 ! Chlamydia
 
+[Term]
+id: MONDO:0025622
+name: Charcot-Marie-Tooth disease, axonal, mitochondrial form, 1
+synonym: "CMTMA1" EXACT ABBREVIATION [OMIM:500013]
+xref: OMIM:500013 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:410434]: classify under CMT

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -412091,6 +412151,22 @@ synonym: "immunodeficiency 75" EXACT [OMIM:619126]
 xref: OMIM:619126 {source="MONDO:equivalentTo"}
 is_a: MONDO:0021094 {source="OMIM:619126"} ! immunodeficiency disease
 
+[Term]
+id: MONDO:0030859
+name: COACH syndrome 2
+synonym: "COACH2" EXACT ABBREVIATION [OMIM:619111]
+xref: OMIM:619111 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:412159]: where is generic COACH?

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -412091,6 +412151,22 @@ synonym: "immunodeficiency 75" EXACT [OMIM:619126]
...
+
+[Term]
+id: MONDO:0030860
+name: neuronopathy, distal hereditary motor, type 5c
+synonym: "Dhmn5C" EXACT [OMIM:619112]
+synonym: "HMN5C" EXACT ABBREVIATION [OMIM:619112]
+synonym: "Spinal Muscular Atrophy, Distal, Type 5C" EXACT [OMIM:619112]
+xref: OMIM:619112 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:412168]: classify under dHMN 

consider adding a grouping dHMN5, we have 5A and 5B. btw, 5B may be conflated, can you look into it

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -412099,6 +412175,13 @@ synonym: "osteogenesis imperfecta, TYPE XXI" EXACT [OMIM:619131]
 xref: OMIM:619131 {source="MONDO:equivalentTo"}
 is_a: MONDO:0019019 {source="OMIM:619131"} ! osteogenesis imperfecta
 
+[Term]
+id: MONDO:0030862
+name: COACH syndrome 3
+synonym: "COACH3" EXACT ABBREVIATION [OMIM:619113]
+xref: OMIM:619113 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:412183]: need generic COACH

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -412182,6 +412300,20 @@ synonym: "CTS2" EXACT ABBREVIATION [OMIM:619161]
 xref: OMIM:619161 {source="MONDO:equivalentTo"}
 is_a: MONDO:0007275 {source="OMIM:619161"} ! carpal tunnel syndrome
 
+[Term]
+id: MONDO:0030885
+name: amyotrophic lateral sclerosis 26 with or without frontotemporal dementia
+synonym: "ALS26" EXACT ABBREVIATION [OMIM:619133]
+xref: OMIM:619133 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:412308]: classify under familiar ALS MONDO:0005144

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -412199,6 +412331,19 @@ synonym: "NPHS22" EXACT ABBREVIATION [OMIM:619155]
 xref: OMIM:619155 {source="MONDO:equivalentTo"}
 is_a: MONDO:0002350 {source="OMIM:619155"} ! familial nephrotic syndrome
 
+[Term]
+id: MONDO:0030896
+name: chromosome 13q33-q34 deletion syndrome
+xref: OMIM:619148 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:412338]: classify this more precisely

consider 

 MONDO:0016911  ! partial deletion of the long arm of chromosome 13

if not, at least chromosomal disease

need monochrom!

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -412720,6 +412938,13 @@ synonym: "immunodeficiency 79" EXACT [OMIM:619238]
 xref: OMIM:619238 {source="MONDO:equivalentTo"}
 is_a: MONDO:0021094 {source="OMIM:619238"} ! immunodeficiency disease
 
+[Term]
+id: MONDO:0030982
+name: sulfide:quinone oxidoreductase deficiency
+synonym: "SQORD" EXACT ABBREVIATION [OMIM:619221]
+xref: OMIM:619221 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:412946]: inborn error of metabolism

also add axiom linking to GO:0070224

File: src/ontology/mondo-edit.obo [hunk 8]
```diff
@@ -412736,6 +412983,41 @@ synonym: "SPGF53" EXACT [OMIM:619258]
...
+synonym: "KTZSL" EXACT ABBREVIATION [OMIM:619229]
+xref: OMIM:619229 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+
+[Term]
+id: MONDO:0030991
+name: bile acid conjugation defect 1
+synonym: "BACD1" EXACT ABBREVIATION [OMIM:619232]
+xref: OMIM:619232 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:412998]: guess there is no BACD2?

File: src/ontology/mondo-edit.obo [hunk 9]
```diff
@@ -412769,6 +413073,28 @@ synonym: "NTCP Deficiency" EXACT [OMIM:619256]
...
+synonym: "SHILCA Syndrome" EXACT [OMIM:619260]
+xref: OMIM:619260 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+
+[Term]
+id: MONDO:0031008
+name: nephrotic syndrome, type 24
+synonym: "NPHS24" EXACT ABBREVIATION [OMIM:619263]
+xref: OMIM:619263 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:413096]: NPHS

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:410434]: done

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:412159]: It looks like OMIM has not created a PS yet to group the 3 subtypes of COACH syndrome. 

COACH SYNDROME 1; COACH1 https://omim.org/entry/216360
COACH SYNDROME 2; COACH2 https://omim.org/entry/619111
COACH SYNDROME 3; COACH3 https://omim.org/entry/619113

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:412159]: I will create a generic grouping class

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:412168]: I created a grouping class: MONDO_0100350 'neuronopathy, distal hereditary motor, type 5' that is equiv to [Orphanet:139536 Distal hereditary motor neuropathy type 5 ](https://www.ebi.ac.uk/ols/ontologies/ordo/terms?iri=http%3A%2F%2Fwww.orpha.net%2FORDO%2FOrphanet_139536)

Orphanet:139536 was MONDO:equvialentTo MONDO_0013884 'neuronopathy, distal hereditary motor, type 5B' and I changed it to MONDO:subClassOf

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:412183]: MONDO_0100349 'COACH syndrome'

File: src/ontology/mondo-edit.obo (hunk 5 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:412308]: done

File: src/ontology/mondo-edit.obo (hunk 6 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: reclassified as child of 'partial deletion of the long arm of chromosome 13'

File: src/ontology/mondo-edit.obo (hunk 7 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:412946]: done

File: src/ontology/mondo-edit.obo (hunk 8 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:412998]: right, it doesn't look like there is:
https://omim.org/search?index=entry&start=1&limit=10&sort=score+desc%2C+prefix_sort+desc&search=BILE+ACID+CONJUGATION+DEFECT+

File: src/ontology/mondo-edit.obo (hunk 9 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:413096]: done

**Pull request merged**